### PR TITLE
fix: remove text and add animation to social media icons

### LINKED ISSUE #101: Removing the text with social media icons
Having only social media icons is a more subtle approach rather than adding a text with them.
Also, add some zoom animation on hover.
image

Refer to CodePen Link for more context
https://codepen.io/_rohan09_/pen/mdBqMNq

### PR DESCRIPTION
**Pull request by brandong512**
Closes #101 

Changed from png icons to svg icons for better quality animation. Added animation that zooms on icon, when mouse hovers over the icon.

### Desktop View
social-media-icon-animation-larger-stable
---

### Mobile View
Screenshot 2021-12-25 232328
---

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -125,8 +125,13 @@ body {
...
   vertical-align: middle;
-  width: 15px;
-  height: 15px;
+  width: 35px;
+  height: 35px;
+  transition: all 0.5s ease-in-out;
+}
+
+.social-media-logo:hover {
+  transform: scale(1.3);
```
Comment (rohan09-raj) [inline css/main.css:134]: Can we change it to `1.25` ??
The reference was only for context, how the end outcome will look like.
Using values in multiples or in ratio scales is a good practice, like `1.25`, `1.5`, `1.75`.

**Review by rohan09-raj — CHANGES_REQUESTED** [review]
Awesome work, requesting a small change.
Also, please look into why checks are failing.

File: css/main.css (hunk 1 above)
Comment (brandong512) [reply css/main.css:134]: Sure thing I'll change that right away

Comment (brandong512) [reply css/main.css]: Alright just fixed issue with the checks and there aren't anymore failures. Also, changed 1.3 --> 1.25 for the icon scaling. Let me know if there's anything else you want me to change

**Review by rohan09-raj — APPROVED** [review]
Approving the changes. Thanks for the PR.
Congrats on getting your first PR merged to RDS.

**Pull request merged**